packages/backend-core/src/middleware/passport/local.ts

```diff
@@ -33,52 +28,31 @@ export async function authenticate(
   if (!email) return authError(done, "Email Required")
   if (!password) return authError(done, "Password Required")
 
+  // EXISTS
   const dbUser = await users.getGlobalUserByEmail(email)
   if (dbUser == null) {
-    return authError(done, `User not found: [${email}]`)
+    console.info(`user=${email} is could not be found`)
+    return authError(done, INVALID_ERR)
   }
 
-  // check that the user is currently inactive, if this is the case throw invalid
+  // ACTIVE
   if (dbUser.status === UserStatus.INACTIVE) {
+    console.info(`user=${email} is inactive`, dbUser)
     return authError(done, INVALID_ERR)
   }
 
-  // check that the user has a stored password before proceeding
+  // HAS PASSWORD
```

Comment:
Are these comments required? The code looks verbose enough

Reply:
I thought the original comments were slightly too verbose which is why I shortened to the simpler statements. With the new log lines they probably are a bit overkill to have them at all now - removed in https://github.com/Budibase/budibase/pull/9672/commits/46f98f76122f88058b9e1231e3a8ff14dda59829